src/components/Completed.js

```diff
@@ -0,0 +1,18 @@
+import storageManager from './Storage.js';
```

Comment:
like you do for the other class, `storageManager` will be updated to `StorageManager`

Reply:
Thank you! 👍